github/orgs_teams.go

```diff
@@ -114,10 +114,39 @@ func (s *OrganizationsService) GetTeam(ctx context.Context, team int) (*Team, *R
 	return t, resp, nil
 }
 
+// NewTeam represents a team to be created or modified.
+type NewTeam struct {
+
+	// The name of the team (Required.)
+	Name         string   `json:"name"`
```

Comment:
Since there's no blank line after `Name` field, and `Description` doesn't have a comment of its own, this reads as if the comment above is for the whole block. You can make it inline to make it more readable. Also, maybe drop the "The" to shorten it.

```Go
Name         string   `json:"name"` // Name of the team. (Required.)
Description  *string  `json:"description,omitempty"`
...
```